runtime/nvqir/qpp/QppCircuitSimulator.cpp

```diff
@@ -140,11 +139,10 @@ class QppCircuitSimulator : public nvqir::CircuitSimulatorBase<double> {
       zero_state(0) = 1.0;
       state = qpp::kron(state, zero_state);
     } else {
-      qpp::ket initState =
-          qpp::ket::Map(const_cast<std::complex<double> *>(stateData), count);
+      qpp::ket initState = qpp::ket::Map(
+          const_cast<std::complex<double> *>(stateData), (1UL << count));
```

Comment:
Might want to note in the doxygen comment that `count` is the power-of-2 exponent.